source/_static/scripts.js

```diff
@@ -0,0 +1,3 @@
+$(document).ready(function() {
+    $("a.reference.external").attr("target", "_blank");
```

Comment:
Do we need this? I see it more as a usability red flag.

Reply:
OK with our without them (I added a comment, not a request changes, so go for it).